src/stream.rs

```diff
@@ -29,10 +44,15 @@ where
         syn: Option<Packet>,
         socket_events: mpsc::UnboundedSender<SocketEvent<P>>,
         stream_events: mpsc::UnboundedReceiver<StreamEvent>,
-        connected: oneshot::Sender<io::Result<()>>,
+        notify_conn_open: oneshot::Sender<Result<(), conn::Error>>,
```

Comment:
why change the name here? IMO `connected` says the same thing in a more succinct way